src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -77,7 +77,7 @@ internal class SqlServerFhirDataStore : IFhirDataStore, IProvideCapability
 
         public async Task<UpsertOutcome> UpsertAsync(ResourceWrapper resource, WeakETag weakETag, bool allowCreate, bool keepHistory, CancellationToken cancellationToken)
         {
-            await _model.EnsureInitialized();
+            await _model.StartAsync();
```

Comment:
Why should we have to call StartAsync() from the consumer?

Reply:
Good point. Removed in 4e00f98 and c1c9e77.